pkg/database/database_env.go

```diff
@@ -0,0 +1,42 @@
+// Copyright KubeArchive Authors
```

Comment:
How about renaming this file to just environment? As we are already inside the `database` package I think there is no need to reiterate that

Reply:
Done.